app/src/main/java/org/mozilla/focus/widget/DefaultBrowserPreference.java

```diff
@@ -23,13 +25,17 @@ public DefaultBrowserPreference(Context context, AttributeSet attrs) {
         super(context, attrs);
     }
 
-    public boolean shouldBeVisible() {
-        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.N;
-    }
-
     @Override
     protected void onClick() {
-        Intent intent = new Intent(Settings.ACTION_MANAGE_DEFAULT_APPS_SETTINGS);
-        getContext().startActivity(intent);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            Intent intent = new Intent(Settings.ACTION_MANAGE_DEFAULT_APPS_SETTINGS);
+            getContext().startActivity(intent);
+        } else {
+            // This link apparently doesn't need the usual SUMO processing: it's not app specific, but platform specific
+            final String url = "https://support.mozilla.org/kb/make-firefox-default-browser-android?utm_source=inproduct&amp;utm_medium=settings&amp;utm_campaign=mobileandroid";
```

Comment:
What about the utm_ flags. Do we need to change them so that we can see users are coming from Focus? (Let's move this question to the issue)